it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/pubsub/PubsubResourceManager.java

```diff
@@ -394,12 +405,65 @@ private static <T> RetryPolicy<T> retryOnDeadlineExceeded() {
         .build();
   }
 
+  /**
+   * Collects the performance metrics for the pubsub resources.
+   *
+   * @param metrics The pubsub metrics will be populated in this map
+   */
+  public void getMetrics(@NonNull Map<String, Double> metrics) {
```

Comment:
it is odd to have a get method which populates a map and then returns a void. Can you rename this to collectPubSubMetrics or something similar ?

Reply:
Done.